dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/cards/conditional-formatting.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Conditional Formatting
+author: Margarita Zakhodyaeva
+---
+
+# Conditional Formatting
+
+For the Card dashboard item, you can apply conditional formatting to the card's visual elements (like Title, Subtitle, different values) and change the card's background.
+
+![](../../../../images/web-dashboard-card-conditional-formatting.png)
+
+> [!Note]
+> Cards that use [legacy layout](https://docs.devexpress.com/Dashboard/118608/create-dashboards/create-dashboards-on-the-web/designing-dashboard-items/cards/layout#legacy-layout-v162-and-earlier) do not support conditional formatting.
+
+## Create a Format Rule
+
+To create a format rule, open the **Conditional Formatting** section in the dashboard item's [Options](../../ui-elements/dashboard-item-menu.md) menu or in the [data item menu](../../ui-elements/data-item-menu.md). Click "+" to add a new format rule:
+	
+![wed-dashboard-cf-add-rule](../../../../images/wed-dashboard-cf-add-rule.png)
+	
+Specify the data item/card used to calculate a condition in the **Common** section. You can also create a format rule for one data item and apply different settings to the other data item. 
+
+Available settings in the **Common** section depend on the selected **Calculated by** option:
+
+* If you select the **Card** option, specify the card by whose values the format rule is calculated. The format rule applies to the [related card](providing-data.md). The **Value Type** option enables you to specify the card's value type according to which a condition is calculated. The **expression** format condition is an exception to this rule and applies to all cards.
```

Comment:
"The **Value Type** option enables you to specify the card's value type according to which a condition is calculated."

This basically says that the Value Type option specifies a card's value type. Will the reader understand this explanation?